### PR TITLE
dunstctl: add page

### PR DESCRIPTION
**Pull request by ExperiBass**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**
Unknown, dunst v1.7.1

### CODE REVIEW

**Comment by ExperiBass** [comment]
ok there, i fixed it, somehow

sorry bout the earlier one, still learning git, and im sure i accidentally botched something somewhere to do that

**Comment by CleanMachine1** [comment]
The error was probably due to the same branch being used for 2 pages and it having loads of commits that were squashed when the first page was merged

**Comment by ExperiBass** [comment]
> The error was probably due to the same branch being used for 2 pages and it having loads of commits that were squashed when the first page was merged 

Sounds like it, I'll remember to use multiple branches next time

**Review by 258204 — COMMENTED** [review]
Looks good to me after @CleanMachine1's comments are addressed.

Thanks for the page!

**Comment by ExperiBass** [comment]
Aaand done! You're welcome uωu

**Review by 258204 — APPROVED** [review]

**Review by nicokosi — APPROVED** [review]
Looks good.👌

File: pages/linux/dunstctl.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# dunstctl
+
+> Control command for `dunst`.
+> More information: <https://man.archlinux.org/man/community/dunst/dunstctl.1.en>.
```
Comment (CleanMachine1) [inline pages/linux/dunstctl.md:4]:
```suggestion
> More information: <https://manned.org/dunstctl>.
```

**Review by CleanMachine1 — APPROVED** [review]

**Pull request merged**